components/Pages/Project/Review/CardReviewSummary.tsx

```diff
@@ -178,24 +178,34 @@ export const CardReviewSummary = () => {
     setRatingData(reviewsWithPercentualRelevance);
   };
 
+  useEffect(() => {
+    const getGapUser = async () => {
+      const response = await fetch(`https://gapapi.karmahq.xyz/grantees/${address}/is-gap-user`)
+      const data = await response.text()
+      setIsGapUser(data)
+    }
+    getGapUser()
+  }, [])
+
   return (
     <div className="flex flex-col w-full gap-5">
       <div className="flex w-full justify-between items-center">
         <div className="flex gap-2">
           <DynamicStarsReview
             totalStars={1}
             rating={0}
-            setRating={() => {}}
+            setRating={() => { }}
             mode={ReviewMode.READ}
           />
           <h2 className="text-base font-semibold font-['Open Sans'] leading-normal">
             Reviews Summary
           </h2>
         </div>
         {isConnected &&
-        project?.recipient &&
-        address &&
-        !isAddressEqual(project.recipient, address) ? ( // Check if the address is equal to the grant recipient address
+          project?.recipient &&
+          address &&
+          isGapUser === 'true' &&
```

Comment:
@hawkjr97 this comparison does not look good. 

Why would you compare this value as string if it represents boolean?

"true" (string) is not nice
true (boolean) is nice

"false" (string) is not nice
false (boolean) is nice

The typing system should be respected and onboarded for this const